tox.ini

```diff
@@ -26,7 +26,7 @@ commands =
     coverage report --fail-under=100 --show-missing
 # parallel mode: make sure all builds complete before we run this one
 depends =
-    py27,py35,py36,py37,py38,pypy,py27-zodb,pypy-zodb,py27-pure-zodb
+    py27,py35,py36,py37,py38,pypy,pypy3
```

Comment:
py27-pure should remain in here I think

Reply:
Does it add much value given the existence of pypy and pypy3?

Reply:
(Note that it's still in the environment list, just not the parallel coverage dependency list.)

Reply:
I dunno, do we want to run `coverage combine` while `coverage run -p` is still running on py27-pure?

I suppose pypy will cover the pure-Python alternative branches so the coverage number _shouldn't_ drop because of this.  As long as we don't have pypy-specific vs Python-pure-specific branches in the code...

Reply:
> I suppose pypy will cover the pure-Python alternative branches so the coverage number shouldn't drop because of this. 

It didn't, so...

> As long as we don't have pypy-specific vs Python-pure-specific branches in the code…

...I don't think we do. If we do in the future, the coverage number will drop and we can know we need to depend on it or rework the code.